### ISSUE TITLE
chap02 78page 오류문의

### ISSUE THREAD

**Issue by mindang** [gilbutITbook/080289#6]
outputs = torch.tensor(outputs).flatten()
을 
outputs = torch.tensor(outputs)
outputs = torch.argmax(outputs,dim=1)
로 바꿔야 하지 않나 싶습니다.

기존의 outputs은 (6912,)로 바로 다음 코드의 데이터셋 분리에서 1383 , 345개로 나눠집니다.
크로스 엔트로피를 사용하므로 argmax를 사용하여 outputs을 1728개로 맞춰야하지 않나 싶습니다!
